numpyro/util.py

```diff
@@ -115,31 +115,26 @@ def scan(f, a, bs):
         return lax.scan(f, a, bs)
 
 
-def tscan(f, a, bs, fields=(0,)):
+def tscan(f, a, bs, transform):
+    if not transform:
+        def transform(x): return x
```

Comment:
It might be more readable if we create an `identity` function and set `transform=identity` by default.